tempesta_fw/http_parser.c

```diff
@@ -303,6 +301,9 @@ parse_int_a(unsigned char *data, size_t len, const unsigned long *delimiter_a,
 		*acc = *acc * 10 + *p - '0';
 	}
 
+	if (unlikely(p - data == len))
```

Comment:
Double check is not acceptable in this performance critical function. Please rewrite the loop as while() with all necessary checks inside.